redis/src/commands/mod.rs

```diff
@@ -371,6 +372,23 @@ implement_commands! {
         cmd("DEL").arg(key)
     }
 
+    /// Conditionally removes the specified key. A key is ignored if it does not exist.
```

Comment:
since these are new, please add to the docs from which version they're available, and phrase it in a way that clarifies that it's only on Redis.